### ISSUE TITLE
log4j issue

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/phenol#45]
Application code gets this message. Any idea how to deal with this in log4j2.xml or properties?
```
No appenders could be found for logger (org.monarchinitiative.phenol.io.obo.OboParser).
log4j:WARN Please initialize the log4j system properly.
```

**Comment by yy20716** [comment]
Peter, I guess we can close this issue if you didn't experience any other log4j issues from the merged one. Thank you.

**Comment by pnrobinson** [comment]
@yy20716 
I am not sure if this is an issue with phenol, but when I use phenol in an application such as this
https://github.com/monarch-initiative/hpoannotqc
I  get this message from the logger
```
$ java -jar target/HpoAnnotQc.jar big-file
log4j:WARN No appenders could be found for logger (org.monarchinitiative.phenol.io.obo.OboParser).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```
I have not been able to figure out how to suppress this warning. 
I have tried to suppress like this but with no success
```
<?xml version="1.0" encoding="UTF-8"?>
<Configuration status="error">
    <Appenders>
        <Console name="Console"
                 target="SYSTEM_OUT">
            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p %t [%t] (%F:%L) - %m%n" />
        </Console>


        <File name="MyLog"
              append="true"
              immediateFlush="true"
              fileName="HpoAnnotQc.log">
            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p %t [%t] (%F:%L) - %m%n" />
        </File>

        <Console name="AntiLog"
                 target="SYSTEM_OUT"/>
    </Appenders>

    <Loggers>
        <Logger name="org.monarchinitiative.hpoannotqc.Main" level="error">
            <AppenderRef ref="Console" level="trace"/>
            <AppenderRef ref="MyLog" level="trace"/>
            <!-- <AppenderRef ref="MyRollingFile" /> -->
        </Logger>

        <Logger name="org.monarchinitiative.phenol">
            <AppenderRef ref="AntiLog" level="off"/>
        </Logger>

        <Root level="trace">
            <AppenderRef ref="Console" level="error"/>
            <AppenderRef ref="MyLog" level="error"/>
            <!-- <AppenderRef ref="MyRollingFile" additivity="false" /> -->
        </Root>
    </Loggers>
</Configuration>
```

**Comment by yy20716** [comment]
Thanks for letting me know. I am currently checking what I can do about this issue. Thanks for your patience.

**Comment by yy20716** [comment]
Peter, it seems that this problem arises from hpoannotqc. I updated the pom.xml of hpoannotqc to fix this issue and confirmed that no error messages arise anymore (I tested this based on the develop branch of phenol and the master branch of hpoannotqc). I tried to commit the updated file but it seems that I don't have the access for th hpoannotqc repo. I only changed pom.xml for hpoannotqc, so you might just replace the pom file with the one in [this gist](
https://gist.github.com/yy20716/f25d01a1382055f24d20948dd90a2073). Thank you.
